process.exit(0)

error("Process should have exited successfully")
